patches/node/api_delete_deprecated_fields_on_v8_isolate.patch

```diff
@@ -0,0 +1,20 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@electronjs.org>
+Date: Wed, 5 Nov 2025 12:50:16 -0500
+Subject: Delete deprecated fields on v8::Isolate
+
+https://chromium-review.googlesource.com/c/v8/v8/+/7081397
```

Comment:
Not a blocker, but this patch should have a "This patch should be upstreamed to Node.js and then removed" type paragraph